testgrid/config/config.yaml

```diff
@@ -0,0 +1,809 @@
+default_test_group:
+  days_of_results: 14
+  tests_name_policy: 2
+  column_header:
+  - configuration_value: Commit
+  use_kubernetes_client: true
+  is_external: true
+default_dashboard_tab:
+  open_test_template:
+    url: https://k8s-gubernator.appspot.com/build/<gcs_prefix>/<changelist>
+  file_bug_template:
+    url: https://github.com/kubernetes/kubernetes/issues/new
+    options:
+    - key: title
+      value: 'E2E: <test-name>'
+    - key: body
+      value: <test-url>
+  attach_bug_template:
+  results_text: See these results in Gubernator
+  results_url_template:
+    url: https://k8s-gubernator.appspot.com/builds/<gcs_prefix>
+  code_search_url_template:
+    url: https://github.com/kubernetes/kubernetes/compare/<start-custom-0>...<end-custom-0>
+test_groups:
```

Comment:
This looks fantastic! Will you please add some comments teaching people what default_test_group, default_dashboard_tab and test_groups do and how to configure them?

Also please add some whitespace between big sections.